### DISCUSSION TITLE
Words not properly loading when typing Khmer language

### DISCUSSION THREAD

**Discussion by KimhengMok** [notaapp/talk#310] [Questions]
Hello, I have an issue and a request.

I really love your minimalistic design which makes it just right for all the things that I need daily for capturing, writing, and reviewing our notes. (It's like Apple Notes with the bi-directional linking ability).

I used Khmer language but it's not working as normal (you'll see the attached video as the reference).

–

One more thing is I have no idea of coding, but I wish you can create a command prompt that link the files or webpages with our note on Nota app without going to its original file directory.
Since I often use hook to connect webpages and files and link it with notes, this would greatly help ease the process and workflow. (I seem to see Obsidian has the ability to do this)

I will also include the reference image as well.

Thank you so much.

https://user-images.githubusercontent.com/84358917/190917314-317f747e-7424-4f0e-9676-2ad44405844c.mp4

![Hook Request](https://user-images.githubusercontent.com/84358917/190917323-3713731e-a2be-4b2c-b53b-75a7a7eb64e0.png)

**Comment by astoilkov** [comment]
Hi,

About the Khmer language: I did take a look at the issue. This is a bug. Our editor doesn't support this type of ligatures available in the Khmer language. I'm working on fixing this but it involves a fundamental change in our editor and it's not a quick fix.

About Hook: I did take a look at this. I think I can quickly fix this. I will try to do so next week.

Sorry for the late response! I was deep in implementing something hard.

**Reply by astoilkov** [reply]
Hook support should land in the next release (due in the first weeks of next year).
